react-components/src/components/GiftCardOptions/giftCardOptions.js

```diff
@@ -48,13 +49,21 @@ const messages = defineMessages({
 });
 
 const GiftCartOptions = props => {
-    const { sku } = props;
+    const { sku, showAddToWishList } = props;
     const [
         giftCardState,
-        { changeAmountSelection, changeCustomAmount, changeOptionValue, changeQuantity, canAddToCart, addToCart }
+        {
+            changeAmountSelection,
+            changeCustomAmount,
+            changeOptionValue,
+            changeQuantity,
+            canAddToCart,
+            addToCart,
+            addToWishlist
+        }
     ] = useGiftCardOptions({ sku });
 
-    const { formatMessage } = useIntl();
+    const intl = useIntl();
```

Comment:
Is this change necessary?

Reply:
It is actually recommended to use `intl` that way to ease extraction. See the https://formatjs.io/docs/tooling/cli example

```
function Comp(props) {
  const {intl} = props
  return intl.formatMessage({
    // The whole `intl.formatMessage` is required so we can extract
    id: 'foo',
    defaultMessage: 'foo',
    description: 'bar',
  })
}
```